src/services/getQuotes.service.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const axios = require('axios');
+const baseURL = 'https://type.fit/api/quotes';
```

Comment:
How will another developer know, that base url s defined in this particular in this line